Doc/library/os.rst

```diff
@@ -3280,16 +3280,21 @@ written in Python, such as a mail server's external command delivery program.
    subprocesses.
 
 
-.. function:: register_at_fork(func, when)
+.. function:: register_at_fork(*, before=None, after_in_parent=None, \
+                               after_in_child=None)
 
-   Register *func* as a function to be executed when a new child process
-   is forked.  *when* is a string specifying at which point the function is
-   called and can take the following values:
+   Register callables to be executed when a new child process is forked
+   using :func:`os.fork`.  The parameters are optional and keyword-only.
```

Comment:
It is impossible and unmaintainable to list them all, I've generalized the wording.